rust/scx_utils/src/topology.rs

```diff
@@ -270,9 +270,9 @@ impl Topology {
     #[cfg(feature = "gpu-topology")]
     pub fn gpus(&self) -> BTreeMap<GpuIndex, &Gpu> {
         let mut gpus = BTreeMap::new();
-        for (_, node) in &self.nodes {
+        for node in self.nodea.values() {
```

Comment:
Typo?

Reply:
yes :)

Reply:
I think I just forgot to test gpu-topology build before committing